### PR TITLE
#3230 Create edit workspace tabs

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3227)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by eechih** [comment]
retry manager

File: ohara-manager/client/src/components/Workspace/Styles.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+export const StyledActions = styled.div(
+  () => css`
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin-bottom: 16px;
+
+    button {
+      margin-left: 8px;
```
Comment (konekoya) [inline ohara-manager/client/src/components/Workspace/Styles.js]: We should probably use `${props.theme.spacing(1)}px`. See https://material-ui.com/customization/spacing/

File: ohara-manager/client/src/components/Workspace/EditWorkspace.js [hunk 2]
```diff
@@ -15,19 +15,95 @@
...
+      aria-labelledby={`edit-workspace-tab-${index}`}
+      {...other}
+    >
+      <Box p={3}>{children}</Box>
+    </Typography>
+  );
+}
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
```
Comment (konekoya) [inline ohara-manager/client/src/components/Workspace/EditWorkspace.js]: isRequired :)

File: ohara-manager/client/src/components/Workspace/Styles.js (hunk 1 above)
Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Styles.js]: I will follow this rule.

File: ohara-manager/client/src/components/Workspace/EditWorkspace.js (hunk 2 above)
Comment (eechih) [reply ohara-manager/client/src/components/Workspace/EditWorkspace.js]: OK

**Pull request merged**